lib/carin_for_blue_button_test_kit/custom_groups/visual_inspection_and_attestation/v200_client/authentication.rb

```diff
@@ -0,0 +1,30 @@
+module CarinForBlueButtonTestKit
+  class CARIN4BBAuthTest < Inferno::Test
+    title 'Uses appropriate authentication'
+    description <<~DESCRIPTION
+      The Health IT Module must
+      - Direct communication through authenticated, authorized, and secure channels.
+      - Use the SMART App Launch Framework’s standalone launch.
+    DESCRIPTION
+    id :c4bb_authentication
+
+    verifies_requirements 'hl7.fhir.us.carin-bb_2.0.0@1',
+                          'hl7.fhir.us.carin-bb_2.0.0@48'
+
+    run do
+      identifier = SecureRandom.hex(32)
+      wait(
+        identifier:,
+        message: <<~MESSAGE
+          I attest that the Health IT Module
```

Comment:
Not a big deal, but I noticed that some say "I attest" and others say "The developer of the Health IT Module", is that intentional? If not, should we make them consistent?

Reply:
yes it's intentional.

"I attest" means the test proctor is able to witness it (visual inspection)

"The developer of the Health IT Module attests" means the test proctor can't witness it (or it would be too difficult), so has to answer the question based on what the developer claims. (attestation)

It's kind of hand-wavy judgement for which gets what.